util/config.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// ConfigDuration is an abstraction used to simplifying unmarshalling
+// json into structs that contain time.Duration fields
+type ConfigDuration struct {
```

Comment:
Maybe rename to `JsonDuration`?

Please document the expected marshaled format as well (a mention to `time.ParseDuration` would be good enough, IMO).